python/dgl/nn/pytorch/conv/gcn2conv.py

```diff
@@ -170,7 +170,7 @@ def set_allow_zero_in_degree(self, set_value):
         """
         self._allow_zero_in_degree = set_value
 
-    def forward(self, graph, feat, feat_0):
+    def forward(self, graph, feat, feat_0, edge_weight=None):
```

Comment:
In ":math: `\beta` ensures that the decay of", the rendering of beta is not correct for the built documentation. Can you also fix that?

Reply:
Fixed